source/elements/dpcpp/source/index.rst

```diff
@@ -49,151 +48,99 @@ some additional extensions being required, and some DPC++ extensions no longer
 required if covered by newer C++ or SYCL versions directly.
 
 .. table:: DPC++ Extensions Table: Support requirements for DPC++
-           implementations above SYCL 1.2.1
+           implementations above SYCL 2020
    :name: Extensions Table
 
    ===========================  ========================  ====================  ====================  ====================  =============
    Feature                      Where defined             CPU                   GPU                   FPGA                  Test [#test]_
    ===========================  ========================  ====================  ====================  ====================  =============
-   Accessor simplifications     `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   bit_cast                     `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Deduction guides             `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Device specific queries      `SYCL 2020 provisional`_  Not required [#tmp]_  Not required [#tmp]_  Not required [#tmp]_  NA [#na]_
-   Extended atomics             `SYCL 2020 provisional`_  Required [#ea]_       Required [#ea]_       Not required [#tmp]_  NA [#na]_
-   Kernel func type attributes  `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   In-order queues              `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Math array                   `SYCL 2020 provisional`_  Not required [#tmp]_  Not required [#tmp]_  Not required [#tmp]_  NA [#na]_
-   Optional lambda name         `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Queue shortcuts              `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Required work-group size     `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Standard layout relaxed      `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Unified Shared Memory        `SYCL 2020 provisional`_  Required [#USM]_      Required [#USM]_      Required [#USM]_      `usm <https://github.com/intel/llvm/tree/sycl/sycl/test/usm>`__
    `Accessor properties`_       DPC++ extension           Required [#aprop]_    Required [#aprop]_    Required [#aprop]_    NA [#na]_
+   `Device-side assert`_        DPC++ extension           Required              Required              Required              NA [#na]_
```

Comment:
I heard recently that this extension is not enabled by default.  (I think you need to predefine a macro to enable it.)  If that is the case, should it still be in this list?

Reply:
Removed it